### DISCUSSION TITLE
Is it possible to set multiple app key when using flutter sdk?

### DISCUSSION THREAD

**Discussion by lmagniez-sd** [pendo-io/pendo-mobile-sdk#190] [Q&A] [answered]
The documentation for the web sdk specify that is is possible to send data to multiple applications. (https://support.pendo.io/hc/en-us/articles/360032201951-Send-data-to-multiple-applications)

> If you have a direct web implementation of Pendo, you can send event data from a single application to multiple Pendo applications that may or may not be in the same subscription by including additionalApiKeys:[] in your install script. 
> ```
> pendo.initialize({
>     additionalApiKeys: ['<API_KEY_FOR_APP2>', '<API_KEY_FOR_APP3>', '<API_KEY_FOR_APP4>'],
>     visitor: {...},
>     account: {...}
> });
> ```

Is there an equivalent when using flutter SDK?

**Comment by MaximShnirman** [comment, marked as answer]
hi @lmagniez-sd, no it is not possible on the Fluter SDK (or any mobile SDK)

**Reply by lmagniez-sd** [reply]
@MaximShnirman 
Thank you for the quick reply. 
Do you have any plan of implementing this feature in the near future?

**Reply by MaximShnirman** [reply]
not as far as i know at this time
